src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion_inpaint_legacy.py

```diff
@@ -21,6 +21,7 @@
 import PIL
 from packaging import version
 from transformers import CLIPFeatureExtractor, CLIPTextModel, CLIPTokenizer
+import torchvision
```

Comment:
`torchvision` is not a hard-dependency. We sadly cannot just import it here and should not use it in this script

Reply:
Ah okay, resizing can also be done with [torch interpolate](https://pytorch.org/docs/stable/generated/torch.nn.functional.interpolate.html). I will resubmit PR with the change